documentation/book/proc-changing-kafka-user.adoc

```diff
@@ -49,37 +55,46 @@ spec:
         operation: Read
 ----
 +
-Or edit the `KafkaUser` resource using {ProductPlatformName} tools.
+. Update the `KafkaUser` resource in {ProductPlatformName}.
++
 ifdef::Kubernetes[]
-On {KubernetesName} this can be done using `kubectl edit`:
-[source,shell,subs=+quotes]
-kubectl edit kafkauser _<your-user-name>_
+On {KubernetesName} this can be done using `kubectl apply`:
 +
+[source,shell,subs=+quotes]
+kubectl apply -f _<your-file>_
 endif::Kubernetes[]
-On {OpenShiftName} this can be done using `oc edit`:
++
+On {OpenShiftName} this can be done using `oc apply`:
 +
 [source,shell,subs=+quotes]
-oc edit kafkauser _<your-user-name>_
+oc apply -f _<your-file>_
++
+. Use the updated credentials from the `my-user` secret in your application.
 
-. Update the `KafkaUser` resource in {ProductPlatformName}.
+[start=1]
+
+.Editing the `KafkaUser` resource using {ProductPlatformName} tools
```

Comment:
Why do we have this added label in the procedure?  We might want to ask the mod docs team how to handle a procedure with 2 options.

Reply:
@ncbaratta, we have two ways of editing a user. Hence, I have used the label to differentiate the same. I will send an email to the mod-docs list to check how to handle a procedure with 2 options.

Reply:
If it helps, we can just keep the `kubectl apply` / `oc apply` procedure. The `kubectl edit` / `oc edit` is applicable everywhere else as well but I think we list it only here.

This is a bit related to my rants for some other PRs ... the users know how to use this and the will anyway ignore the procedure and do `oc create` / `oc replace` / `oc apply` / `oc edit` based on what they are used to.